docs/components/inputs/IntergerInput.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 6
+---
+
+# IntergerInput
+
+Provides an input field for integer values, validating that user input it's a valid integer, and showing error if not.  
+Shows by default a small button to multiply input's value \* 10^18 to transform to wei.
+
+![IntegerInput Example](/img/integerInput.png)
+
+## Import
+
+```ts
+import { IntegerInput } from "~~/components/scaffold-eth";
+```
+
+## Usage
+
+```ts
+const [txValue, setTxValue] = useState<string | bigint>("");
+
+<IntegerInput
+  value={txValue}
+  onChange={(updatedTxValue) => {
+    setDisplayedTxResult(undefined);
```

Comment:
We can remove this `setDisplayedTxResult`

Reply:
Updated 🙌